app/models/concerns/stat/activity_history.rb

```diff
@@ -2,59 +2,49 @@ class Stat < ApplicationRecord
   module ActivityHistory
     extend ActiveSupport::Concern
 
-    DEFAULT_STATS = {
-      'total' => 0,
-      'activity' => []
-    }.freeze
-
-    def recalculate!
-      activity = user.library_events.by_kind(media_column)
-                     .eager_load(media_column)
-
-      # clear stats_data
-      self.stats_data = {}
-      self.stats_data = {
-        total: activity.count,
-        activity: activity # collection of events
+    def default_data
```

Comment:
how come you removed `week_high_score` and `total_progress`?

Also, because I don't feel like actually testing your code does it return days in this format?
```
    # Example of days stats:
    # 'days' => {
    #   '2017' => {
    #     '08' => {
    #       '20' => {
    #
    #       }
    #     }
    #   }
    # }
```

Reply:
Nope, changed the format to `{ '2017-08-20' => {} }` since it's waaaaaay easier for d3 to process

Reply:
oh okay, and the first question?

Reply:
Neither one belongs in this Stat.  For now I've completely removed "best week" functionality since it's not a launch feature and the logic was really big, but I'll bring that back out when we're ready for it.  The total_progress is now in AmountConsumed, where it belongs